Library/Formula/procmail.rb

```diff
@@ -0,0 +1,88 @@
+class Procmail < Formula
+  desc "procmail: sort incoming mail into separate folders/files"
```

Comment:
You don't need the formula name in here. It'll be prepended automatically.

```ruby
desc "Sort incoming mail into separate folders/files"
```

is fine.